src/main/resources/edu/cornell/kfs/sys/cu-spring-sys.xml

```diff
@@ -35,6 +35,7 @@
 				<value>documentReindexerJob</value>
 				<value>autoCancelBatchJob</value>
 				<value>kimFeedJob</value>
+                <value>kimFeedV2Job</value>
```

Comment:
Formatting looks odd on this line.

Reply:
Formatting should be fixed now.; I was using spaces but I've now converted them into tabs for consistency with the adjacent lines.

At some point, we may want a user story (or several user stories) to convert tabs into spaces throughout our code.

Reply:
Thank you